frontend/src/app/pages/Data/Features/NationalMap/deckgl-layers.ts

```diff
@@ -145,7 +146,7 @@ export function renderLayers(
   if (!settings) return;
 
   const geo =
-    settings.geography.includes(Geography.COUNTY) &&
+    settings.geography == Geography.COUNTY &&
```

Comment:
Updated this to only show one "map" at a time. I think we can probably further simplify this by hiding the company and facility type toggles until we're showing the "all facilities" option.